rust/crd/src/security.rs

```diff
@@ -426,12 +436,19 @@ impl KafkaTlsSecurity {
     }
 
     /// Creates ephemeral volumes to mount the `SecretClass` into the Pods
-    fn create_tls_volume(volume_name: &str, secret_class_name: &str) -> Result<Volume, Error> {
+    fn create_tls_volume(
+        kafka_name: &str,
```

Comment:
The variable name should show that a service name is expected, e.g. `kafka_role_service_name`.

Reply:
Renamed to `kafka_bootstrap_service_name`